package.json

```diff
@@ -9,8 +9,9 @@
     "test": "jest",
     "start": "NODE_ENV=production node server.js",
     "format-staged": "pretty-quick --staged --no-restage --bail",
-    "format-check": "pretty-quick --check --verbose",
-    "lint-check": "eslint src pages"
+    "format-check": "prettier --check '{.,pages/**,src/**,static/**}/*.{js,jsx,md}'",
+    "lint-check": "eslint src pages",
+    "format": "prettier --write '{.,pages/**,src/**,static/**}/*.{js,jsx,md}'"
```

Comment:
let's make it `format-all` as we discussed?

Reply:
Yep. Done in 7aaa8aa0.